administrator/components/com_schemaorg/src/Controller/DisplayController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ShazmaSiddiqui\Component\Schemaorg\Administrator\Controller;
```

Comment:
pointer: component should be under the `Joomla\Component` namespace

Reply:
done